### ISSUE TITLE
bug: expected `()`, found `Extensions` in command.rs for v1.14.0

### ISSUE THREAD

**Issue by chungquantin** [r0gue-io/pop-cli#317]
This happens with both `assets-parachain` and `contracts-parachain` v1.14.0.

https://github.com/user-attachments/assets/5c87d33a-8740-493e-a400-5d419df4bd23

**Comment by brunopgalvao** [comment]
Can you try generating a new assets or contracts parachain using `main` and running `pop build`?

**Comment by AlexD10S** [comment]
The fix of this was merged into `main`. We'll generate a release with the bug fix soon. 
In the meantime, to unblock you, you can install `pop-cli` directly from GitHub: cargo install --locked --git https://github.com/r0gue-io/pop-cli
